### DISCUSSION TITLE
Test version event annotation is not displayed

### DISCUSSION THREAD

**Discussion by Terracotta-62** [artisan-roaster-scope/artisan#1148] [Q&A]
![9cab3be16914b49963fcb7698502acc](https://user-images.githubusercontent.com/116416332/232174219-33555aa1-9a5e-4499-93a3-0218801035e1.png)
![993d4f9f069e6c253890eefe3210a21](https://user-images.githubusercontent.com/116416332/232174231-240f1f19-9440-4013-ba6f-4d56a4902294.png)

**Comment by MAKOMO** [comment]
Thanks for your report. This should be fixed now. New continuous builds are just being created. Note that those continuous builds are not fully tested and sometimes not even start. We just did a major refactoring of the whole code base which for sure resulted in many minor and maybe even major issues like the one you just reported. We are now in the process to test and iron out all such issues we can catch. If you find more strange behaviours, please report.
